proposals/0265-unlifted-datatypes.rst

```diff
@@ -126,10 +126,10 @@ data type explicitly *may* have levity polymorphic kind
 
 ::
 
-  data List a :: TYPE (BoxedRep LiftedRep) where
+  data List a :: TYPE (BoxedRep Lifted) where
     Nil :: List a
     Const :: a -> List a -> List a
-  data SList a :: TYPE (BoxedRep UnliftedRep) where
+  data SList a :: TYPE (BoxedRep Unlifted) where
     Nil :: SList a
```

Comment:
s/`Nil`/`SNil`/ ? Especially since you define an alternative version of `SList` as `data SList a = SNil | SCons a (List a)` below.